proto/frostdb/snapshot/v1alpha1/snapshot.proto

```diff
@@ -13,16 +12,7 @@ message FooterData {
 
 message Table {
   string name = 1;
-  message TableConfig {
-    oneof schema {
-      frostdb.schema.v1alpha1.Schema deprecated_schema = 1;
-      frostdb.schema.v1alpha2.Schema schema_v2 = 2;
-    }
-    int64 row_group_size = 3;
-    int64 block_reader_limit = 4;
-    bool disable_wal = 5;
-  }
-  TableConfig config = 2;
+  frostdb.table.v1alpha1.TableConfig config = 5;
```

Comment:
I believes this can stay as field number 2 (we don't care about backwards compatibility yet)

Reply:
That's fair. I tried to "do the right thing" but you're probably right.